extensions/github1s/src/adapters/ossinsight/constants.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @file trending constants
+ * @author netcon
+ */
+
+export enum RankingPeriod {
+	Today = 'Today',
+	ThisWeek = 'ThisWeek',
+	ThisMonth = 'ThisMonth',
+}
+
+export const RankingLanguages = [
```

Comment:
where does this data of RankingLanguages taken from? any more possibilities to add (or is there something provided by ossinsights team)?

Reply:
I think, it is from OSSInsights object